reservation/src/test/java/io/skai/reservation/mapper/TicketMapperTest.java

```diff
@@ -0,0 +1,30 @@
+package io.skai.reservation.mapper;
+
+import io.skai.reservation.dto.TicketDto;
+import io.skai.reservation.jooq.tables.pojos.Ticket;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+
+import java.math.BigDecimal;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class TicketMapperTest {
+    private final Ticket TICKET = new Ticket(1L, 1L, 1L, new BigDecimal(120), "1B");
```

Comment:
let's have:

1. some business values, not 1 1 1
2. why do you need to use `1L` instead of `1`?